README.md

```diff
@@ -16,10 +16,11 @@
 
 ## Description
 
-**FIXME:** Ensure the *Description* section is correct and complete, then remove this message!
-
 Minimally manage a Kubernetes cluster! This module will configure the core
-services required to get a basic Kubernetes cluster up and running.
+services required to get a basic Kubernetes cluster up and running, based on the
+RedHat distribution available in the `extras` CentOS repo.
```

Comment:
This doesn't parse for me. You should probably say something like 'based on the Red Hat provided packages'. Mixing RHEL and CentOS will confuse people.